### PR TITLE
Edit index file

### PR DESCRIPTION
**Pull request by Angham116**
#3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+config.js
```
Comment (kannanhassouna) [inline .gitignore:2]: add white space here

**Review by Angham116 — COMMENTED** [review]
done

File: public/index.html [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+	<title>Github Info</title>
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+    crossorigin="anonymous" />
 </head>
 <body>
-
+	<div class="main">
+		<form class="main__form">	
+			<input class="main__form--input" type="text" placeholder="plz enter sth">
+			<button class="main__form--submit"><i class="far fa-search"></i></button>
```
Comment (knanahassouna1) [inline public/index.html]: Indentation !!

Comment (kannanhassouna) [inline public/index.html:12]: use BEM 
http://getbem.com/introduction/

File: public/index.html [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
+		</form>
+	</div>
 </body>
```
Comment (kannanhassouna) [inline public/index.html:15]: add white space between head , body and html tags

**Pull request merged**